### PR TITLE
NMS-16221: Update Helm Charts readme file

### PR DESCRIPTION
**Pull request by Bonrob2**
NMS-16221: Update Helm Charts readme file - updated the introductory content in the Helm Charts readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,165 @@
...
 
 Note that this is one way to approach the solution.
 We recommend that you study the content of the Helm chart and tune it for your needs.
 
-## Charts
-* [Horizon](horizon/README.md)
-* [Minion](minion/README.md); This is an experimental chart.
+For more information about deploying in a containerized environment, including requirements and external dependencies, refer to [Containerized Deployment](https://docs.opennms.com/horizon/latest/deployment/core/containers.html) in the main product documentation.
+
+If you are not already familiar with Horizon or Meridian, we recommend reviewing our [product documentation](https://docs.opennms.com/horizon/32/deployment/core/introduction.html) and our [Horizon](https://www.opennms.com/horizon/) or [Meridian](https://www.opennms.com/meridian/) websites.
```
Comment (mershad-manesh) [inline README.md:12]: Should we point the user to the Meridian documentation?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,165 @@
...
+| sentinel.resources.requests.cpu | string | `"2"` |  |
+| sentinel.resources.requests.memory | string | `"2Gi"` |  |
+| sentinel.terminationGracePeriodSeconds | int | `60` |  |
+| timezone | string | `"America/New_York"` |  |
+
+----------------------------------------------
+Autogenerated from chart metadata using [helm-docs v1.11.3](https://github.com/norwoodj/helm-docs/releases/v1.11.3)
+
+## Additional charts
+* [Minion](minion/README.md). This is an experimental chart.
```
Comment (mershad-manesh) [inline README.md:165]: We have duplicate entry for Minion;

File: README.md [hunk 3]
```diff
@@ -1,13 +1,165 @@
...
+
+## Version compatibility
+
+| Helm chart version | Horizon version(s) | Meridian version(s) |
+| ----------- | ----------- | ----------- |
+| 1.x | Horizon 32.x | Meridian 2023.x |
+
+## Values
+
+| Key | Type | Default | Description |
```
Comment (mershad-manesh) [inline README.md:33]: I think this table is created automatically; We shouldn't hardcode it

File: README.md (hunk 1 above)
Comment (Bonrob2) [reply README.md:12]: I guess so. They are identical, but I can see why we would want to provide Meridian as well.

File: README.md [hunk 4]
```diff
@@ -1,13 +1,10 @@
...
-Each deployment through OpenNMS Helm Charts has a single Core server, Grafana, and a custom Ingress that shares the RRD files and some configuration files, and multiple Sentinels for flow processing.
+If your organization uses Kubernetes or Red Hat OpenShift, OpenNMS makes a Helm chart available to simplify deployment of Horizon and Meridian.
 
 Note that this is one way to approach the solution.
 We recommend that you study the content of the Helm chart and tune it for your needs.
 
 ## Charts
-* [Horizon](horizon/README.md)
-* [Minion](minion/README.md); This is an experimental chart.
+* [Core](horizon/README.md)
```
Comment (mershad-manesh) [inline README.md:9]: We can keep `Horizon` here

File: horizon/README.md.gotmpl [hunk 5]
```diff
@@ -1,33 +1,24 @@
-# OpenNMS Helm Charts -- Horizon
+# OpenNMS Helm Charts
```
Comment (mershad-manesh) [inline horizon/README.md.gotmpl:1]: Lets keep `-- Horizon` in the readme file for now

File: horizon/README.md.gotmpl [hunk 6]
```diff
@@ -1,33 +1,24 @@
...
 
+```
 helm repo add opennms https://opennms.github.io/helm-charts
 
 helm install monms opennms/horizon --set domain=domain1.com  --create-namespace
 ```
-
-## Product documentation
-For information on requirements, installation, manual configuration, and troubleshooting tips, see the [product documentation](https://docs.opennms.com/opennmshelmcharts/latest/installation/introduction.html#requirements).
+NOTE: For Meridian, repalce "horizon" in the second command with "meridian".
```
Comment (mershad-manesh) [inline horizon/README.md.gotmpl:21]: we don't need this line.. `horizon` is the chart name;

**Review by mershad-manesh — CHANGES_REQUESTED** [review]
overall looks good, we just need a few tweaks.

File: README.md (hunk 4 above)
Comment (Bonrob2) [inline README.md:9]:
```suggestion
* [Horizon](horizon/README.md)
```

**Review by mershad-manesh — APPROVED** [review]

**Pull request merged**